### PR TITLE
refactor(ui5-input): refactor RTL in DatePicker, Menu, (Multi)ComboBox,  RadioButton, Select, SegBtn, TabContainer and Token(izer) components - part 5

### PR DESCRIPTION
**Pull request by ilhan007**
Reactor RTL styles with CSS Logical props. The change affects:

Topic B
- DatePicker
- Menu
- SegmentedButton

Topic P
- Select
- BarcodeScannerDialog

Topic RD
- TabContainer
- RadioButton

Topic RL
 - MessageStrip
 - (Multi)ComboBox, Token, Tokenizer,
 - TableColumn, TableGroup
 
Note: 
- the BarcodeScannerDialog does not need to set dir anymore as it depends on the Dialog and the DIalog haw been refactored already.
- the MessageStrip has been refactored previously, but the dir has not been removed from the template.

### CODE REVIEW

**Review by tsanislavgatev — DISMISSED** [review]

**Review by nnaydenow — APPROVED** [review]
BarcodeScannerDialog and Select look good for me.

File: packages/main/src/themes/TabContainer.css [hunk 1]
```diff
@@ -126,10 +126,4 @@
...
 :host([media-range="XL"]) .ui5-tc__content {
 	padding: 1rem 3rem;
-}
-
-/*** RTL ***/
-[dir=rtl] .ui5-tc__overflow {
-	margin-right: auto;
-	margin-left: 0.25rem;
-}
+}
```
Comment (gmkv) [inline packages/main/src/themes/TabContainer.css:129]: In `/packages/main/src/themes/TabInStrip.css`, could you also change the following lines:

```css
.ui5-tab-expand-button > [ui5-button] {
	height: 1.5rem;
	min-width: 1.5rem;
	margin-top: 0.25rem;
-       margin-left: 0.625rem;
+       margin-inline-start: 0.625rem;
 }

       . . .

 .ui5-tab-single-click-icon {
-       margin-left: 0.875rem;
+       margin-inline-start: 0.875rem;
 }
```

**Review by gmkv — CHANGES_REQUESTED** [review]
RadioButton looks good. I have a suggestion for TabContainer.

**Review by gmkv — APPROVED** [review]
LGTM for Topic RD

**Review by niyap — APPROVED** [review]

**Pull request merged**